redeem/gcodes/M2x.py

```diff
@@ -326,8 +321,7 @@ def process_gcode(self, fn, g):
                 self.printer.processor.execute(file_g)
 
             self.printer.sd_card_manager.current_lock.acquire()
-            self.printer.sd_card_manager.current_file = None
-            self.printer.sd_card_manager.current_file_count = None
+            self.printer.sd_card_manager.current_byte_count = self.printer.sd_card_manager.current_file_size
```

Comment:
I interpreted that the command "closed" the file after printing is complete. But looking back at how other systems define it, this is the correct behavior. Seems strange to me that there's no way to "close" the file and set the state of the machine back to where it was when it first started.